docs/pages/usage/failures.md

```diff
@@ -8,39 +8,36 @@ pagination_label: Service failures
 
 # Failures
 
-The methods that are used in `make` may fail. In order to more informatively provide information about this outside the service, the following methods were prepared.
+In a simple use case, all service failures (or crashes) will come from input, output or internal attributes.
```

Comment:
I think in the english version we can skip "(or crashes)". This clarification was appropriate in the Russian version.

Reply:
Yes, you're absolutely right, boss. Only russian version needs this clarification